packages/i18n/locales/en/translation.json

```diff
@@ -1,4 +1,25 @@
 {
+  "actionCategory": {
+    "authorizationsDescription": "Grant and revoke authorizations, or perform an authorized execution.",
```

Comment:
> Grant, revoke, and perform authorizations.

We can make this shorter and less confusing. You either know what you're doing in this category or you don't.